### DISCUSSION TITLE
organization github

### DISCUSSION THREAD

**Discussion by fcomte** [InseeFrLab/legacy-onyxia-entrypoint#58] [Polls]
Onyxia is software initially incubated in the [Insee](https://www.insee.fr)'s innovation team. 
As an innovation project, it was first published in the [InseeFrLab](https://github.com/InseeFrLab) organization.
There are evidences that this software is used and developed by other organizations. We are thinking that it could be interesting to migrate and leave this incubating organization to reflect that Onyxia has matured enough.

Should we move the Onyxia project to another organization?
If so, should we choose [InseeFr](https://github.com/InseeFr) organization to express that Insee is still strongly involved in this project or should we create a dedicated organization?

**Comment by trygu** [comment]
Thank you for your great work!
